tasks/create-cert-webroot.yml

```
@@ -0,0 +1,30 @@
+---
+- name: Check if certificate already exists.
+  stat:
+    path: /etc/letsencrypt/live/{{ cert_item.domains | first }}/cert.pem
+  register: letsencrypt_cert
```

```suggestion
  register: _certbot_letsencrypt_cert
```

Private variables should be prefixed with `_certbot_`